packages/core/admin/admin/src/utils/setOpacity.js

```diff
@@ -0,0 +1,6 @@
+const setOpacity = (hex, alpha) =>
```

Comment:
We should put it in helper plugin and rename it (something like setHexOpacity maybe)